src/shared/components/mutationTable/column/CosmicColumnFormatter.tsx

```diff
@@ -132,7 +184,8 @@ export default class CosmicColumnFormatter {
                 <span className={styles['cosmic-table']}>
                     <b>{value}</b> occurrences of <b>{cosmic[0].keyword}</b>{' '}
                     mutations in COSMIC
-                    <CosmicMutationTable data={cosmic} />
+                    <CosmicMutationTable data={cosmic} columns={columns} />
+                    <b>Last Updated: Before 2017</b>
```

Comment:
@gblaih not sure if there's some hacky way to see if the data is different from what we have in the db for public/msk portals. It's possible local installers have an updated version of COSMIC